userService.js

```diff
@@ -0,0 +1,130 @@
+const mssql = require("mssql/msnodesqlv8");
+const dbConfig = require("./dbconfig");
+const sqlHelper = require("./userServiceSQLHelper");
+
+const pool = new mssql.ConnectionPool(dbConfig.config);
+
+function paginateUsers(req, res) {
+    pool.connect(() => {
```

Comment:
Opening new connection on each requests is not a good idea. It limits amount of active users by possible DB connection amount

Reply:
What approach/direction should I take?